Makefile

```diff
@@ -57,6 +57,4 @@ db-down-dev:
 db-down-test:
 	docker rm -f spi_test
 
-reset: db-down-dev db-up-dev migrate
-	@# reset dev db (test db is automatically reset by tests)
-
+db-reset: db-down db-up migrate
```

Comment:
The README said `db-reset` but the Makefile said `reset`, so I picked `db-reset` 👍

Reply:
Also, when I ran this locally it consistently hung at the `migrate` stage if *only* the dev db had been reset. Resetting both made it work, and there should be no downside to it.

Reply:
I think that may be because the db is not quite ready and the migrate kicks in too soon. There is a way to wait on it but it’s probably not worth fiddling that in tbh.